### PR TITLE
Konveyor Addon Pattern

### PR DESCRIPTION
**Pull request by ariveroi**
*Issue #, if available:*

*Description of changes:*

Created based on this PR: https://github.com/aws-samples/cdk-eks-blueprints-patterns/pull/91

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/konveyor.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+```sh
+make deps
+make build
+make pattern konveyor deploy
+```
+
+When deployment completes, the output will be similar to the following:
+
+<img src="./addon/src/assets/img.png"/>
```
Comment (elamaran11) [inline docs/patterns/konveyor.md:97]: Where is this image, i dont see in PR?

File: package.json [hunk 2]
```diff
@@ -36,10 +39,8 @@
...
     "@snyk-partners/snyk-monitor-eks-blueprints-addon": "^1.1.0",
     "aws-cdk": "2.88.0",
-    "@aws-sdk/client-eks": "^3.316.0",
     "eks-blueprints-cdk-kubeflow-ext": "0.1.9",
-    "source-map-support": "^0.5.21",
-    "@instana/aws-eks-blueprint-addon": "^1.0.4"
+    "source-map-support": "^0.5.21"
   },
   "overrides": {
     "@aws-quickstart/eks-blueprints": "1.10.1",
```
Comment (elamaran11) [inline package.json:46]: Make sure blueprints version is updated

File: package.json [hunk 3]
```diff
@@ -25,8 +25,11 @@
   },
   "dependencies": {
     "@aws-quickstart/eks-blueprints": "1.10.1",
```
Comment (elamaran11) [inline package.json:27]: Make sure blueprints version is updated

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@ariveroi Thankyou for taking time to submit a PR. Im confused if its a Partner Addon or an open source addon. We have some gaps as highlighted below :
1. Is this an open source product or a Partner product. Blueprints Partner or Open Source Addon is missing in cdk-blueprints-repo. Please check our [extendability](https://aws-quickstart.github.io/cdk-eks-blueprints/extensibility/#public-extensions) guide for Public/private/partner extensions. These two PRs needs to be submitted in parallel for review.
2. Do we have any license for this product if so, please share it with us.
3. If this is a partner product, do we have confirmation from partner to support ongoing manitenance of the addon, if so, please have the partner confirm in both PRs.
4. For this PR Readme update is missing for `make list` command and other spot
5. `mkdocs.yml` update is missing for the addon documentation.

File: docs/patterns/konveyor.md (hunk 1 above)
Comment (ariveroi) [reply docs/patterns/konveyor.md]: fixed, added the image to the images folder

File: package.json (hunk 2 above)
Comment (ariveroi) [reply package.json]: updated

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@ariveroi Thankyou for taking time to update the PR. We have some gaps as highlighted below :

- Do we have any license for this product if so, please share it with us.
- The doc is missing secret manager command to add secret to AWS environment as highlighted.
- Do we have confirmation from partner to support ongoing maintenance of the addon, if so, please have the partner confirm in both PRs.
- Make sure you pull the code from main because we had some breaking change going in yesterday

**Comment by cbellone** [comment]
hello @elamaran11

This is Celestino Bellone, I am an employee of Claranet Switzerland, and I am one of the maintainers of https://github.com/claranet-ch/konveyor-eks-blueprint-addon .

As requested, this is to confirm that Claranet will support the ongoing maintenance of the Konveyor EKS Addon.

Let me know if you need anything else from our side.

Thanks,
Celestino

**Comment by elamaran11** [comment]
> hello @elamaran11
> 
> This is Celestino Bellone, I am an employee of Claranet Switzerland, and I am one of the maintainers of https://github.com/claranet-ch/konveyor-eks-blueprint-addon .
> 
> As requested, this is to confirm that Claranet will support the ongoing maintenance of the Konveyor EKS Addon.
> 
> Let me know if you need anything else from our side.
> 
> Thanks, Celestino

Thankyou @cbellone Appreciate your notes on the PR. This is enough for us to proceed with further review. We will work with @ariveroi to help him work on the comments and get this merged asap.

File: docs/patterns/konveyor.md [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+make deps
+make build
+make pattern konveyor deploy
+```
+
+When deployment completes, the output will be similar to the following:
+
+<img src="./images/konveyor-cdk-output.png"/>
+
+## Example configuration
```
Comment (elamaran11) [inline docs/patterns/konveyor.md:103]: This example configuration should be removed in this repo because you already have a pattern code.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@ariveroi Minor change. If this already peer validated by Riccardo, can you ask him to comment on the PR on the peer validation

**Review by freschri — APPROVED** [review]

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**